app.py

```diff
@@ -336,18 +399,16 @@ def group_category(group=[], category='all'):
         if group == 'press-centre':
             group = 'canonical-announcements'
 
-        groups = _get_groups_by_slug(slugs=group)
+        groups = _get_group_by_name(group) #_get_groups_by_slug(slugs=group)
```

Comment:
Please remove comment

Reply:
`_get_group_by_name` returns a single group so this variable should be called `group` to stop it being confusing.